src/lib/parts.ts

```diff
@@ -128,6 +132,14 @@ export class AttributePart implements Part {
   }
 }
 
+/**
+ * A Part that controls a location within a Node tree. Like a Range, NodePart
+ * has start and end locations and can set and update the Nodes between those
+ * locations.
+ *
+ * NodeParts support several value types: primitives, Nodes, TemplateResults,
```

Comment:
Don't they also support other NodeParts? I thought that was how repeat was implemented.

Reply:
A nested NodePart isn't a value set into another NodePart with setValue, it's actually just placed into the correct DOM position.